Libraries/LibGUI/Clipboard.cpp

```diff
@@ -79,10 +100,18 @@ void Clipboard::set_data(const StringView& data, const String& type)
     WindowServerConnection::the().send_sync<Messages::WindowServer::SetClipboardContents>(shared_buffer->shbuf_id(), data.length(), type);
 }
 
+void Clipboard::set_data(const RefPtr<Gfx::Bitmap> bitmap, const String& type)
```

Comment:
`RefPtr` here implies that passing null is a valid action, but it's not.
Instead, let's use `const Gfx::Bitmap&` to indicate that a non-null bitmap is required, and that this function does not take ownership. It's basically never required to pass a smart pointer as an argument. If the callee wants to take ownership of something, it can instantiate a `RefPtr` or `NonnullRefPtr` from a `T*` or `T&` at will :)

Also, why is the `type` parameter needed here? Do we need different types of bitmaps somehow?

Reply:
You're right. I'm just getting my hand on managing refs + ptrs and I was a bit lost when I made that. Now I understand clearly what you're saying (thanks, books).

Type is now fixed to bitmap so we match the IPC call at WindowServer.ipc:63